SCClassLibrary/Common/Control/Server.sc

```diff
@@ -741,13 +741,13 @@ Server {
 
 	sendDefaultGroups {
 		defaultGroups.do { |defGrp|
```

Comment:
looks good. While you are at it, could you please replace the needless abbreviation `defGrp` by either `defaultGroup`, `group`, or something really non-distractingly generic like `x` or `each`?

Reply:
yes, please, sorry for the slangyness

Reply:
you are welcome :)

Reply:
done!